packages/back-end/src/models/DataSourceModel.ts

```diff
@@ -162,10 +163,11 @@ export async function createDataSource(
     projects,
   };
 
-  // Test the connection and create in the database
   await testDataSourceConnection(datasource);
   const model = await DataSourceModel.create(datasource);
 
+  await queueCreateInformationSchema(datasource, organization);
```

Comment:
Depending on how we want to launch this, this might make sense to put behind a feature flag, along with the front-end changes.

Reply:
Alternatively, we can merge this pr when ready and all that will happen is for new Postgres and BigQuery datasources that are created, we will quietly create informationSchemas for them. But, the user won't have any idea until we merge the front-end [PR](https://github.com/growthbook/growthbook/pull/1050).